cheroot/workers/threadpool.py

```diff
@@ -151,12 +151,20 @@ def __init__(
             server (cheroot.server.HTTPServer): web server object
                 receiving this request
             min (int): minimum number of worker threads
-            max (int): maximum number of worker threads
+            max (int): maximum number of worker threads (-1 for no max)
             accepted_queue_size (int): maximum number of active
                 requests in queue
             accepted_queue_timeout (int): timeout for putting request
                 into queue
+
+        Raises:
+            ValueError: if the min/max values are invalid
         """
+        if min < 1:
+            raise ValueError('min must be > 0')
+        if -1 < max < min:
+            raise ValueError('max must be > min (or -1 for no max)')
```

Comment:
```suggestion
            raise ValueError(f'{max=} must be > {min=} (or -1 for no max)')
```

Reply:
This `{x=}` f-string syntax is only available in Python3.8+ and this package supports v3.6 - I've used the more verbose `x={x}`syntax instead.